internal/chrome/chrome_darwin.go

```diff
@@ -30,6 +30,9 @@ func (s *CookieStore) getKeyringPassword(useSaved bool) ([]byte, error) {
 	case `chrome`:
 		service = `Chrome Safe Storage`
 		account = `Chrome`
+	case `chromium`:
```

Comment:
I haven't tried it, but I understand that they are all based on `chromium` and their principles should be the same.